src/datetime/addHours.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * To add hours to a datetime.
+ * @param {Date} dateFrom
+ * @param {number} hours
+ * @return Date
+ */
+
+export const addHours = (dateFrom: Date, hours: number): Date => {
+  let ret = new Date();
+  ret.setTime(dateFrom.getTime() + hours * 3600000);
```

Comment:
Consider using constants instead of using the literal value `3600000`, or maybe a function `hoursFromMilliseconds`. It would be easier to read.

Reply:
Thanks @mcrispino, I went for the second option.

export const addHours = (dateFrom: Date, hours: number): Date => {
  return new Date(dateFrom.getTime() + hoursToMilliseconds(hours));
};